grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -840,9 +843,11 @@ class OrderService {
                     orderItem.recipient = recipient ? personService.getPersonByNames(recipient) : null
 
                     def estReadyDate = null
+                    Locale locale = LocalizationUtil.currentLocale
+                    SimpleDateFormat readyDateFormat = new SimpleDateFormat(LocalizationUtil.getLocalizedOrderImportDateFormat(locale))
```

Comment:
I am not sure about this part. What if someone changed a locale after doing export and while filling in the template with dates that follow the format that is in the header? Should we catch this case and throw an exception that the current locale does not match the one that the export was made on? Or we don't care about it and rely on training? Feels like this **might** cause some confusion and support issues and perhaps this is not the best approach.

Reply:
I feel like we should not cover this part. As I told you, getting the info about header in this method would be a pain in the neck, hence I assumed that if someone is using `es_MX`, he/she would use the same template for export and import. 

On the other hand we can say - ok, if we coverered this edge case, then what if someone uses `es_MX`, expects to import the date in the `dd/MM/yyyy` format, but unfortunately the header contains `MM/dd/yyyy` - what should we prioritize then - the template's header or the actual locale? My mind says the locale is more important that the header :thinking:

Reply:
I would say that the header is more important in this case because it contains the format that is meant to be used in the column. 

But yeah I agree feels like this might cause more headaches than it is required and what you have is enough as a quick solution.

Reply:
>I would say that the header is more important in this case because it contains the format that is meant to be used in the column.
+1